docs/configure-cms.md

```diff
@@ -2,60 +2,57 @@
 
-Important: some of shop elements will not load properly if your database credentials are not correct, such as the add to cart button on the product page.
+### Register account on the game server
 
-### - Register account on the game server
-
-Wordpress by default does not allow the registration from any user so you first need to enable the registration going to `Settings -> General` and check next to `Membership` the option `Anyone can register`.
+Wordpress by default does not allow the registration from any user, so you first need to enable the registration going to `Settings -> General` and check next to `Membership` the option `Anyone can register`.
 
 Now, when a user will register on your website, after the **verification email process**, the CMS will register an account on the game server.
 
 For the **verification email process** there are several different options:
 
-- install a **Mail Server** in your webserver (you can find a lot of guides online)
-- Install the plugin **WP SMTP** via `Admin Panle -> Plugins -> Add new` and **[configure it with gmail or other SMTP server](https://www.wpbeginner.com/plugins/how-to-send-email-in-wordpress-using-the-gmail-smtp-server/)**, you just need to have a mail like GMAIL and connect it to WP .
-- Remove the verification email process
-
-For the last option you have to:
-
-- install and activate the plugin **Auto Login New User After Registration**
-- go to `Settings -> Auto Login New User After Registration`
-- check the checkbox next to `Check to ENABLE Password required fields on new user registration form`
+a. install a **Mail Server** in your webserver (you can find a lot of guides online)
+b. install the plugin **WP SMTP** via `Admin Panle -> Plugins -> Add new` and **[configure it with gmail or other SMTP server](https://www.wpbeginner.com/plugins/how-to-send-email-in-wordpress-using-the-gmail-smtp-server/)**, you just need to have a mail like GMAIL and connect it to WP .
 
-With this, the users do not need to confirm their emails and the game account will be created istantly.
 I recommend you to install the plugin `Manage Notification E-mails` to manage in detail the email sending from your website to the users or to the admins.
 
-**If you get some error during the registration of the user see below the troubleshooting section.**
+If you encounter errors, please see the [Troubleshooting](##Troubleshooting) section below for more details.
 
-### - Enable the shop
+### Enable the shop
 
 To add the shop in your website selling items and services you have to:
 
-- install and configure WooCommerce plugin using `Admin Panel -> Plugins -> Add new` (you can skip the configuration and remove all the shipment options).
+1. install and configure WooCommerce plugin using `Admin Panel -> Plugins -> Add new` (you can skip the configuration and remove all the shipment options).
   Now the pages like `Cart, Shop, Checkout` will be created and added into your menu, you can edit them from `Pages`.
```

Comment:
to keep the newline in the markdown preview you have to put `two empty spaces` at the end of the line, be careful about it (check all file in the github preview)